spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/DefaultSource.scala

```diff
@@ -23,10 +23,10 @@ import DefaultSource._
  *      CREATE TEMPORARY TABLE tmpTable
  *      USING org.apache.spark.sql.cassandra
  *      OPTIONS (
- *       c_table "table",
+ *       table "table",
  *       keyspace "keyspace",
  *       cluster "test_cluster",
- *       push_down "true",
+ *       pushdown "true",
  *       spark_cassandra_input_page_row_size "10",
  *       spark_cassandra_output_consistency_level "ONE",
```

Comment:
It wasn't really tested, because I removed the settings, yet the tests pass. So the tests definitely were *not* verifying these settings being in effect.